src/balloonpanel/balloonpanelview.js

```diff
@@ -305,43 +301,55 @@ function getAbsoluteRect( elementOrRangeOrRect ) {
 	return absoluteRect;
 }
 
-/**
- * Returns the client rect of the element limited by the visible (to the user)
- * viewport of the browser window.
- *
- *		[Browser viewport]
- *		+---------------------------------------+
- *		|                        [Element]      |
- *		|                        +----------------------+
- *		|                        |##############|       |
- *		|                        |##############|       |
- *		|                        |#######^######|       |
- *		|                        +-------|--------------+
- *		|                                |      |
- *		+--------------------------------|------+
- *		                                 |
- *		                                  \- [Element rect visible in the viewport]
- *
- * @private
- * @param {HTMLElement|Object} element Object which visible area rect is to be determined.
- * @returns {AbsoluteDomRect} An absolute rect of the area visible in the viewport.
- */
-function getAbsoluteRectVisisbleInTheViewport( element ) {
-	const limiterRect = new AbsoluteDomRect( element, 'limiterElement' );
-
-	const windowScrollX = window.scrollX;
-	const windowScrollY = window.scrollY;
-	const bodyWidth = document.body.clientWidth;
-	const bodyHeight = document.body.clientHeight;
+// Returns the client rect of the element limited by the visible (to the user)
+// viewport of the browser window.
+//
+//		[Browser viewport]
+//		+---------------------------------------+
+//		|                        [Element]      |
+//		|                        +----------------------+
+//		|                        |##############|       |
+//		|                        |##############|       |
+//		|                        |#######^######|       |
+//		|                        +-------|--------------+
+//		|                                |      |
+//		+--------------------------------|------+
+//		                                 |
+//		                                  \- [Element rect visible in the viewport]
+//
+// @private
+// @param {HTMLElement|Object} element Object which visible area rect is to be determined.
+// @returns {AbsoluteDomRect} An absolute rect of the area visible in the viewport.
+function getAbsoluteRectVisibleInTheViewport( element ) {
+	const limiterRect = getAbsoluteRect( element );
```

Comment:
I think we should call it `elementRect`. Function is used for the limiter element only but it should feel more like an utility.